### PR TITLE
Select text modal adjustments 140

### PR DESCRIPTION
**Pull request by httpjamesm**
Addresses #140

### CODE REVIEW

**Comment by httpjamesm** [comment]
@aeharding We should be good to go here

**Comment by aeharding** [comment]
Thanks for these adjustments, I'll try to review as soon as I can

File: src/features/comment/Comment.tsx [hunk 1]
```diff
@@ -197,6 +198,8 @@ export default function Comment({
   // Comment from slice might be more up to date, e.g. edits
   const comment = commentById[commentView.comment.id] ?? commentView.comment;
 
+  const [selectTextModalIsOpen, setSelectTextModalIsOpen] = useState(false);
```
Comment (aeharding) [inline src/features/comment/Comment.tsx:201]: Can we avoid putting this in Comment.tsx?

Perhaps we can resolve this with #169

Comment (aeharding) [reply src/features/comment/Comment.tsx:201]: (this component is already doing way too much)

Comment (httpjamesm) [reply src/features/comment/Comment.tsx:201]: Hmm. I suppose we can put this out and change its value with React context. How does that sound?

**Comment by aeharding** [comment]
This PR is superseded by #513 so closing. Thank you!

**Pull request closed without merging**